templates/push.html

```diff
@@ -5,4 +5,4 @@
 {%- endif -%}
 \[{{repo_name}}] \[{{repo}}/{{branch}}] {{user}} pushed {{commits}} {{commit}}:
 {% for c in commit_messages %}
-  - [{{c.hash}}]({{c.url}}) {{c.msg}}{% endfor %}
+  - {{c.url}} {{c.msg}}{% endfor %}
```

Comment:
I kind of like having the hash be a [markdown link][1] to the url. I also realize two things:

a. Not everyone uses backends that render markdown by default, and not every backend renders things in the same way.
b. Preferred message format changes from person to person.

This is exactly what I was addressing in this pending PR errbotio/errbot#879 , that I submitted a while back. If will allow users to override templates without changing a plugin's code.

[1]: https://daringfireball.net/projects/markdown/syntax#link